docs/source/cookbooks/override-operator-images.md

```diff
@@ -0,0 +1,27 @@
+# Override operator images
+
+## Full deploy override
+In order to override images during an actual deploy, you have to:
+
+* clone [openstack-k8s-operators/openstack-operator](https://github.com/openstack-k8s-operators/openstack-operator/)
+* edit [custom-bundle.Dockerfile](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/custom-bundle.Dockerfile) in order to point to the wanted images
+* run `go mod edit` in the [project root](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/go.mod) and update references
+* run `go mod edit` in the [apis directory](https://github.com/openstack-k8s-operators/openstack-operator/blob/main/apis/go.mod#L11) and update references
+
+Then, you'll have to pass down the repository path to the ci-framework, rebuild it, and deploy:
+```YAML
+cifmw_operator_build_meta_src: /path/to/openstack-operator
+cifmw_operator_build_meta_build: true
+cifmw_operator_build_push_registry: quay.io
```

Comment:
Maybe is worthy to add a note here about the internal OpenShift regsitry, for those that doesn't have an external registry available.

Reply:
right - updated.